datafusion/physical-plan/src/aggregates/row_hash.rs

```diff
@@ -643,6 +674,21 @@ impl GroupedHashAggregateStream {
     }
 }
 
+fn find_sort_options(
+    output_ordering: Option<&LexOrdering>,
+    expr: &dyn PhysicalExpr,
+) -> SortOptions {
+    if let Some(ordering) = output_ordering {
+        for e in ordering {
+            if e.expr.as_ref().dyn_eq(expr) {
+                return e.options;
+            }
+        }
+    }
+
+    SortOptions::default()
+}
+
```

Comment:
I wasn't 100% sure about this implementation. Perhaps there's a better (more elegant) way to figure out what the correct `SortOptions` are.